dashboard/config/linux/bits/android-5.10.yml

```diff
@@ -32,4 +32,5 @@ config:
 
  # Android has back-ported the KASAN_STACK_ENABLE => KASAN_STACK rename from v5.11
  - KASAN_STACK_ENABLE: [n, override]
- - KASAN_STACK: [override]
+ # Doesn't boot on Cuttlefish with KASAN_STACK
+ - KASAN_STACK: [override, -cuttlefish]
```

Comment:
dashboard/config/linux/android14-5.15-cuttlefish.config **currently has** "CONFIG_KASAN_STACK=y"
Will it boot?

PS:
dashboard/config/linux/android-5.10-cuttlefish.config **currently has** # CONFIG_KASAN_STACK is not set

Reply:
Thanks for pointing this out; android-5.10 does now boot with Cuttlefish! There was a recent change to the Cuttlefish boot size since additional KASAN parameters were causing an overflow, which may have been the source? Either way I've removed this.